### DISCUSSION TITLE
HA Warning:  sensor_power_meter_reactive-power_2 is using native unit 'kVarh'

### DISCUSSION THREAD

**Discussion by TMoersch** [wlcrs/huawei_solar#259] [Setting up the integration] [answered]
Hallo, habe gerade eben gesehen das HA im home-assitent.log eine Warnung herausgibt.
```WARNING..... entity sensor_power_meter_reactive_power_2 is using native unit of measurement 'kVarh' which ist not a valid unit for device class ('reactive_power') it is using```.
Ich sehe zwar keine Fehlreaktion in der Integration aber vielleicht nutzt es dennoch jemanden als Hinweis.

Habe noch eine Frage...
In Fusion Solar werden seit einiger Zeit die Batterie-Packs einzeln dargestellt und vor allem die Temperatutwerte der Batterie-Packs.
Der SoC über alles reicht mir eigentlich, aber die Temperaturwerte hätte ich gerne in der Überwachung. Kann man die Sensorabfragen eigentlich selbst ergänzen, vorrausgesetzt man wüsste an welcher Stellen man die Register abfragen müsste?
Wirklich cool wäre wenn man auch die laufende SW-Versionen sehen könnte....

soory kann es nicht löschen, habe aber alles unter bugs gefunden, vlt. kann es ja ein Admin löschen.

Thomas

**Comment by wlcrs** [comment]
I don't speak sufficiently German to read your message. Please use English.

**Comment by TMoersch** [comment, marked as answer]
problem ist solved with HA Core Update und the new Version V1.2.1
